### PR TITLE
docs(plugin): refactor limit-conn.md

### PR DESCRIPTION
**Pull request by lunarwhite**
### Description

update documents under `docs/zh/latest/plugins/` along with `docs/en/latest/plugins/`

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/plugins/limit-conn.md [hunk 1]
```diff
@@ -189,28 +168,39 @@ curl --include \
...
 </html>
-````
+```
+
+## Disable Plugin
+
+To disable the `limit-conn` Plugin, you can delete the corresponding JSON configuration from the Plugin configuration. APISIX will automatically reload and you do not have to restart for this to take effect.
+
+```shell
+curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (SylviaBABY) [inline docs/en/latest/plugins/limit-conn.md:192]:
```suggestion
curl http://127.0.0.1:9180/apisix/admin/routes/1  \
-H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

File: docs/zh/latest/plugins/limit-conn.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ---
 title: limit-conn
+keywords:
+  - APISIX
+  - API 网关
+  - Limit Connection
+description: 本文介绍了 Apache APISIX limit-conn 插件的相关操作，你可以使用此插件限制对你的服务的并发请求数。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/limit-conn.md:7]:
```suggestion
description: 本文介绍了 Apache APISIX limit-conn 插件的相关操作，你可以使用此插件限制服务的并发请求数。
```

File: docs/zh/latest/plugins/limit-conn.md [hunk 3]
```diff
@@ -95,17 +102,17 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
-你也可以通过 web 界面来完成上面的操作，先增加一个 route，然后在插件页面中添加 limit-conn 插件：
-![enable limit-conn plugin](../../../assets/images/plugin/limit-conn-1.png)
-
 ## 测试插件
 
-上面启用的插件的参数表示只允许一个并发请求。 当收到多个并发请求时，将直接返回 503 拒绝请求。
+按上述配置启用插件后，服务只允许一个并发请求，当收到多个并发请求时，将直接返回 `503` HTTP 状态码，拒绝请求。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/limit-conn.md:107]:
```suggestion
按上述配置启用插件后，服务将只允许一个并发请求；当收到多个并发请求时，将直接返回 `503` HTTP 状态码，拒绝请求。
```

File: docs/zh/latest/plugins/limit-conn.md [hunk 4]
```diff
@@ -115,36 +122,15 @@ curl -i http://127.0.0.1:9080/index.html?sleep=20
...
-```
-
-现在就已经移除了 limit-conn 插件了。其他插件的开启和移除也是同样的方法。
-
 ## 限制 WebSocket 连接的并发数
 
 Apache APISIX 支持 WebSocket 代理，我们可以使用 `limit-conn` 插件限制 WebSocket 连接的并发数。
 
-1、创建路由并启用 WebSocket 代理和 `limit-conn` 插件
+1、创建路由并启用 WebSocket 代理和 `limit-conn` 插件：
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/limit-conn.md:129]:
```suggestion
1、创建路由并启用 WebSocket 代理和 `limit-conn` 插件。
```

File: docs/zh/latest/plugins/limit-conn.md [hunk 5]
```diff
@@ -169,7 +155,7 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 
 上述路由在 `/ws` 上开启了 WebSocket 代理，并限制了 WebSocket 连接并发数为 1，超过 1 个并发的 WebSocket 连接将返回 `503` 拒绝请求。
 
-2、发起 WebSocket 请求，连接建立成功
+2、发起 WebSocket 请求，返回 `101` HTTP 状态码表示连接建立成功：
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/limit-conn.md:158]:
```suggestion
2、发起 WebSocket 请求，返回 `101` HTTP 状态码表示连接建立成功。
```

File: docs/zh/latest/plugins/limit-conn.md [hunk 6]
```diff
@@ -209,3 +185,22 @@ Server: APISIX/2.15.0
 </body>
 </html>
 ```
+
+## 移除插件
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/limit-conn.md:189]:
```suggestion
## 禁用插件
```

File: docs/zh/latest/plugins/limit-conn.md [hunk 7]
```diff
@@ -23,28 +28,29 @@ title: limit-conn
...
+| rejected_code      | string  | 否 | 503    | [200,...,599]                                                                             | 当请求数超过 `conn` + `burst` 阈值时，返回的 HTTP 状态码。                                                                                                                                                                |
+| rejected_msg       | string | 否                                |            | 非空                                          | 当请求数超过 `conn` + `burst` 阈值时，返回的响应体。                                                                                                                                                                     |
+| allow_degradation              | boolean  | 否                                | false       |                                                                     | 当插件功能临时不可用时是否允许请求继续。当设置为 `true` 时，启用插件降级并自动允许请求继续。                                                                                                                                                    |
```
Comment (hf400159) [inline docs/zh/latest/plugins/limit-conn.md:45]:
```suggestion
| allow_degradation              | boolean  | 否                                | false       |                                                                     | 当设置为 `true` 时，启用插件降级并自动允许请求继续。                                                                                                                                                    |
```

**Review by SylviaBABY — APPROVED** [review]

File: docs/zh/latest/plugins/limit-conn.md [hunk 8]
```diff
@@ -95,17 +102,17 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
-你也可以通过 web 界面来完成上面的操作，先增加一个 route，然后在插件页面中添加 limit-conn 插件：
-![enable limit-conn plugin](../../../assets/images/plugin/limit-conn-1.png)
-
 ## 测试插件
 
-上面启用的插件的参数表示只允许一个并发请求。 当收到多个并发请求时，将直接返回 503 拒绝请求。
+按上述配置启用插件后，服务将只允许一个并发请求；当收到多个并发请求时，将直接返回 `503` HTTP 状态码，拒绝请求。
```
Comment (tzssangglass) [inline docs/zh/latest/plugins/limit-conn.md:107]:
```suggestion
按上述配置启用插件后，在这条路由资源上，APISIX 将只允许一个连接；当有更多连接进入时，APISIX 会直接返回 `503` HTTP 状态码，拒绝连接。
```

**Review by SylviaBABY — APPROVED** [review]

**Review by hf400159 — DISMISSED** [review]

**Review by tzssangglass — APPROVED** [review]

File: docs/en/latest/plugins/limit-conn.md [hunk 9]
```diff
@@ -99,19 +100,17 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
-<!-- ![enable limit-conn plugin](https://raw.githubusercontent.com/apache/apisix/master/docs/assets/images/plugin/limit-conn-1.png) -->
-
 ## Example usage
 
 The example above configures the Plugin to only allow one concurrent request. When more than one request is received, the Plugin will respond with a 503 status code:
```
Comment (tzssangglass) [inline docs/en/latest/plugins/limit-conn.md:105]: also need to change English

**Review by tzssangglass — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**